src/Cell.js

```diff
@@ -41,6 +32,6 @@ export default class Cell extends Component {
   }
 }
 
-Cell.propTypes = {
-  productRow: PropTypes.object
-}
+// Cell.propTypes = {
+//   productRow: PropTypes.object
+// }
```

Comment:
don't leave commented code in! Also: why did you comment it out in the first place?